ahref-frontend/src/app/shared/sdk/models/Seat.ts

```diff
@@ -59,6 +61,10 @@ export class Seat implements SeatInterface {
           name: 'column',
           type: 'number'
         },
+        "prva": {
```

Comment:
Inappropriate naming convention makes code less readable. Stick to english words that semantically describe what the variables and functions represent.